src/redux/sagas/assistantSaga.jsx

```diff
@@ -459,7 +459,7 @@ function formatTelegramLink(url) {
   let urlType = matchPattern(url, KNOWN_LINK_PATTERNS);
   if (urlType !== KNOWN_LINKS.TELEGRAM) {
     throw new Error(
-      "formatTelegramLink: Expected telegram link but got " + urlType
+      "formatTelegramLink: Expected telegram link but got " + urlType,
```

Comment:
Why the additional comma?

Reply:
I think this must be from the linter.

Reply:
Can remove if required.

Reply:
Shouldn't be much of a difference either way so I'll leave it up to you. The rest is looking good.